chimedb/dataflag/orm.py

```diff
@@ -378,6 +378,30 @@ def create_opinion(
         return opinion
 
 
+class DataFlagCategoryType(base_model):
+    """Categories for each day of data.
+
+    Attributes
+    ----------
+    name
+        The name of the category.
+    description
+        A full description of the category.
+    """
+
+    # NOTE: this could almost use the DataSubsetType baseclass except the metadata field
+    # in that isn't really applicable
+    name = pw.CharField(max_length=64, unique=True)
+    description = pw.TextField(null=True)
+
+
+class DataFlagOpinionCategory(base_model):
+    """Many-to-many relationship giving the categories identified for each opinion."""
+
+    opinion = pw.ForeignKeyField(DataFlagOpinion)
+    category = pw.ForeignKeyField(DataFlagCategoryType)
+
+
```

Comment:
```suggestion
    class Meta:
        primary_key = pw.CompositeKey("opinion", "category")

```
I meant to include this suggestion in my review